contrib/svc-cat-apiserver-aggregation-tls-setup.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh --
```

Comment:
I'd prefer to remove the `--` since it'll cause people to wonder what the heck we're up to when it serves no purpose what so ever. Unless we want to just mess with people's heads, then I'm all for it! ;-)